managers/oclcClassify.py

```diff
@@ -59,7 +59,7 @@ def getClassifyResponse(self):
 
         try:
             self.execQuery()
-        except (HTTPError, ReadTimeout):
+        except (ConnectTimeout, HTTPError, ReadTimeout):
```

Comment:
Adding this request exception is a good quality of life to catch errors like this in the future